### PR TITLE
HV-1615 Some typos and improvements to the Dutch translations

### PR DESCRIPTION
**Pull request by Sanne**
Assigned to @bwijsmuller

@bwijsmuller zou jij dit even kunnen naakijken? Mischien de rest van de file ook even controleren, ik gebruik deze taal niet vaak meer :)

 - https://hibernate.atlassian.net/browse/HV-1615

### CODE REVIEW

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_nl.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 javax.validation.constraints.AssertFalse.message        = moet onwaar zijn
-javax.validation.constraints.AssertTrue.message         = moet waar zijn sein
+javax.validation.constraints.AssertTrue.message         = moet waar zijn
 javax.validation.constraints.DecimalMax.message         = moet kleiner dan ${strict == false ? 'of gelijk aan ' : ''}{value} zijn
 javax.validation.constraints.DecimalMin.message         = moet groter dan ${inclusive == true ? 'of gelijk aan ' : ''}{value} zijn
-javax.validation.constraints.Digits.message             = numerieke waarde ligt buiten het toegestane bereik (<{integer} cijfers>,<{fraction} cijfers> verwacht)
```
Comment (bwijsmuller) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_nl.properties]: 'toegestane' was correct so please revert this line change

**Comment by bwijsmuller** [comment]
The rest seems an improvement

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_nl.properties (hunk 1 above)
Comment (Sanne) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_nl.properties]: done, thanks. I was unsure but it looked very odd :)

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request closed without merging**